lib/matplotlib/tests/test_image.py

```diff
@@ -716,15 +705,20 @@ def test_figureimage_setdata():
     assert im._A[0, 0] == 0, 'value changed'
 
 
-def test_pcolorimage_setdata():
+@pytest.mark.parametrize(
+    "image_cls,x,y,a", [
+        (NonUniformImage,
+         np.arange(3.), np.arange(4.), np.arange(12.).reshape(4, 3)),
```

Comment:
This isn't right; you're passing 3 as the `order`.
```suggestion
         np.arange(3.), np.arange(4.), np.arange(12.).reshape((4, 3))),
```

Reply:
Actually, I checked again and this is allowed for the class method.

Reply:
I agree the extra set of parentheses helps legibility, though.